src/Makefile

```diff
@@ -218,11 +215,11 @@ $(REDIS_CHECK_AOF_NAME): $(REDIS_SERVER_NAME)
 
 # redis-cli
 $(REDIS_CLI_NAME): $(REDIS_CLI_OBJ)
-	$(REDIS_LD) -o $@ $^ ../deps/hiredis/libhiredis.a ../deps/linenoise/linenoise.o $(FINAL_LIBS)
+	$(REDIS_LD) -o $@ $^ ../deps/hiredis/libhiredis.a $(JEMALLOC_PATH)/lib/libjemalloc.a ../deps/linenoise/linenoise.o $(FINAL_LIBS)
 
 # redis-benchmark
 $(REDIS_BENCHMARK_NAME): $(REDIS_BENCHMARK_OBJ)
-	$(REDIS_LD) -o $@ $^ ../deps/hiredis/libhiredis.a $(FINAL_LIBS)
+	$(REDIS_LD) -o $@ $^ ../deps/hiredis/libhiredis.a $(JEMALLOC_PATH)/lib/libjemalloc.a $(FINAL_LIBS)
 
 dict-benchmark: dict.c zmalloc.c sds.c siphash.c
```

Comment:
I found that this change got impact for 2 other configuration:
-make dict-benchmark
-make test-sds
They also used jemalloc. Maybe it is good idea to add "$(JEMALLOC_PATH)/lib/libjemalloc.a" in FINAL_LIBS ?

Reply:
Good point, fixed